.circleci/config.yml

```diff
@@ -0,0 +1,43 @@
+version: 2
+
+jobs:
+  build:
+    docker:
+      # The image used to build our project, build
+      # your own using the Dockerfile provided below
+      # and replace here. The official circleci rust 
+      # build was used but this may be replaced.
+      - image: circleci/rust:latest
+
+    environment:
+      # Set your codecov token if your repository is private.
+      #CODECOV_TOKEN: <your-token>
+      #TZ: "/usr/share/zoneinfo/Europe/Paris"
+
+    steps:
+      - checkout
+      - restore_cache:
+          key: project-cache
+      - run:
+          name: Version information
+          command: rustc --version; cargo --version; rustup --version
+      - run:
+          name: Nightly Build
+          command: |
+            rustup toolchain install nightly
+            rustup run nightly rustc --version --verbose
+            rustup run nightly cargo --version --verbose
+            rustup run nightly cargo build
+            rustup target add wasm32-unknown-unknown --toolchain nightly
+
+      - run:
+          name: Test
+          command: ./setup.sh
```

Comment:
This step does the same installation as the "Nightly Build" so its not really needed.
But you called it test, so maybe you meant it to run `cargo test` instead?